R/lifetable_indices.R

```diff
@@ -0,0 +1,105 @@
+lt.check <- function(life_table) {
+
+  # check if life_table is of class "mortaar_life_table"
+
+  if (!inherits(life_table,"mortaar_life_table_list")
+      & !inherits(life_table,"mortaar_life_table"))  {
+    paste0(
+      "The dataset does not seem to be a mortAAR life table."
+    ) %>%
+      message
+  } else if (!inherits(life_table,"mortaar_life_table"))  {
+    paste0(
+      "Please select only one life table."
+    ) %>%
+      message
+  }
+}
```

Comment:
We already have `is.mortaar_life_table_list` and `is.mortaar_life_table`. Do we need this function?

Reply:
Okay, I have changed the code to make use of these already existing functions. The function itself is necessary, in my opinion, because it might not be self-evident for the user that she cannot use the mortaar_life_table_list as input. I am open for any suggestions, however.

Reply:
Ah - I understand. Hm.